### PR TITLE
add anycubic vyper

### PR DESCRIPTION
**Pull request by ThatGuyZim**
Thanks for the heads up nallath, files are as follows

### CODE REVIEW

File: anycubic_vyper_platform.stl [hunk 1]
```diff
@@ -0,0 +1,4286 @@
+solid CATIA STL
```
Comment (nallath) [inline anycubic_vyper_platform.stl:1]: Could you add it as a binary STL (or better; a 3MF)? That helps keeping the size down quite a bit.

**Review by nallath — CHANGES_REQUESTED** [review]

File: anycubic_vyper_platform.stl (hunk 1 above)
Comment (ThatGuyZim) [reply anycubic_vyper_platform.stl]: Uploaded anycubic_vyper_platform.3mf for you

Comment (ThatGuyZim) [reply anycubic_vyper_platform.stl]: can't use .3mf files for some reason, bed positioned wrong in the print area

**Comment by nallath** [comment]
Oh, one more: Is there a specific reason that the .def.json is called "anycubic_vyper_" (and not "anycubic_vyper")?

**Comment by ThatGuyZim** [comment]
I had a few versions of the file before this, you can remove the second underscore

File: anycubic_vyper_.def.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "version": 2,
+  "name": "Anycubic Vyper",
+  "inherits": "fdmprinter",
+  "metadata": {
+    "visible": true,
+    "author": "ThatGuyZim",
+    "manufacturer": "Anycubic",
+    "file_formats": "text/x-gcode",
+    "platform": "anycubic_vyper_platform.3mf",
```
Comment (nallath) [inline anycubic_vyper_.def.json:10]: This still references the 3mf.

Comment (ThatGuyZim) [reply anycubic_vyper_.def.json]: line changed to reference stl instead

**Review by nallath — APPROVED** [review]

**Comment by nallath** [comment]
The files are still in the wrong location, but I will fix that for you. For the next time, they should be located in the resources folder

**Comment by nallath** [comment]
It's merged in: 1740f8e88075c96388206f8b3258228a444f09d4

**Pull request closed without merging**